src/react-query/PersistQueryClientProvider.tsx

```diff
@@ -0,0 +1,60 @@
+// https://github.com/TanStack/query/blob/main/packages/react-query-persist-client/src/PersistQueryClientProvider.tsx
```

Comment:
Had issues where React Query was importing a different instance of the `QueryClientProvider` causing children to not consume the correct `queryClient`. I had a lot of trouble trying to figure out why, so I just copied the source.